mews-api.md

```diff
@@ -685,11 +703,29 @@ There are certain rules that need to be followed in order for Mews to process th
         120,
         120
       ],
+      "amounts": [
```

Comment:
Consider using the same formatting as in Connector API - meaning 4 space indentation. Especially with very nested examples, it's much more readable IMHO.